### PR TITLE
Plain js object

### PR DESCRIPTION
**Pull request by ahzia**
In this project, we had to build a basic website that allows users to add/remove books from a list using javascript objects and arrays.

# General requirements 

- Make sure that there are no linter errors.
- Make sure that you used the correct Github Flow.
- Make sure that you documented your work in a professional way.

# Project requirements 

- Implement only a basic UI with plain HTML
- Create a collection that keeps a list of books (hint: you can use an array of objects for that).
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection (hint: you can use the array filter() method).
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:

1. A new book is added to the collection.
2. The new book is displayed in the page.

- Make sure that when a user clicks the "Remove" button:

1. The correct book is removed from the collection.
2. The correct book dissapears from the page.

- Make sure that data is preserved in the browser's memory by using localStorage.
- To debug your application, you can use console.log(). Remember to remove debugging code before pushing your code to the remote repository.
- Don't use any library or framework, use only plain JavaScript.
- You need to use pair-programming for this project. We will check to see if both partners submitted code by looking at the commit history for each project. Therefore you need to use only one repository for your pair-programming group.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+function addBook() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const error = document.getElementById('error');
+  let status = false;
+  if (title === '') {
+    error.innerHTML = 'title is Required';
+  } else {
+    books.forEach((book) => {
+      if (book.title === title && book.author === author) {
```
Comment (mikenath223) [inline js/main.js]: - [ ] Kindly change all regular functions to es6 arrow functions. As stated in the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.).

**Comment by mikenath223** [comment]
- [x] Kindly update the readme using this [template](https://github.com/microverseinc/readme-template).

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**